ember-app/app/controllers/issue/reference.js

```diff
@@ -6,13 +6,13 @@ var IssueReferenceController = Ember.Controller.extend({
 
   fetchCommit: function(commit){
     var _self = this;
-    var repo = this.get("controllers.issue.model.repo.data.repo.full_name");
+    var repo = this.getRepoName(commit.commit_url);
     return ajax("/api/" + repo + "/commit/" + commit.commit_id)
```

Comment:
Lets add `{global: false}` to this ajax call so any potential 404's don't hit the global handler.